Stripe/StripeiOS/Source/STPAnalyticsClient+BasicUI.swift

```diff
@@ -51,7 +51,7 @@ extension STPPaymentContext {
             }()
             let params: [String: Any] = [
                 "duration": duration,
-                "default_payment_method": defaultPaymentMethod,
+                "selected_lpm": defaultPaymentMethod,
```

Comment:
Will this break existing queries that expect default_payment_method to be populated?

Reply:
No, `default_payment_method` was added earlier this week.